<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationSet</class>
 <widget class="QWidget" name="StationSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProSet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>项目选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbboxProSelect"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>工位选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbboxStationSelect"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>相机选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbboxCamSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>相机步骤</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbboxCamStep">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>功能选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbboxFunctionSelect"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnAddToUse">
         <property name="text">
          <string>添加到使用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnDeleteFromUse">
         <property name="text">
          <string>删除使用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnSave">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>相机使用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabwgtCamUse"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
